docs/operate-and-deploy/migrations-tool.md

```diff
@@ -0,0 +1,609 @@
+    apply                 Migrates the metadata schema to a new schema version.
+    create                Creates a blank migration file with the specified description, which can then be populated with ksqlDB statements and applied as the next schema version.
+    destroy-metadata      Destroys all ksqlDB server resources related to migrations, including the migrations metadata stream and table and their underlying Kafka topics. WARNING: this is not reversible!
+    help                  Display help information
+    info                  Displays information about the current and available migrations.
+    initialize-metadata   Initializes the migrations schema metadata (ksqlDB stream and table) on the ksqlDB server.
+    new-project           Creates a new migrations project directory structure and config file.
+    validate              Validates applied migrations against local files.
+
+See 'ksql-migrations help <command>' for more information on a specific
+command.
+```
+
+The `ksql-migrations` tool supports migrations files containing the following
+types of ksqlDB statements:
+* `CREATE STREAM`
+* `CREATE TABLE`
+* `CREATE STREAM ... AS SELECT` 
+* `CREATE TABLE ... AS SELECT`
+* `CREATE OR REPLACE`
+* `INSERT INTO ... AS SELECT`
+* `TERMINATE <queryID>`
+* `DROP STREAM`
+* `DROP TABLE`
+* `ALTER STREAM`
+* `ALTER TABLE`
+* `INSERT INTO ... VALUES`
+* `CREATE CONNECTOR`
+* `DROP CONNECTOR`
+* `CREATE TYPE`
+* `DROP TYPE`
+* `SET <property>`
+* `UNSET <property>`
+
+Requirements and Installation
+-----------------------------
+
+The `ksql-migrations` tool is available with all ksqlDB versions starting from
+ksqlDB 0.17 (equivalently, Confluent Platform 6.2). You can use the tool to
+manage any ksqlDB cluster running version ksqlDB 0.10 (equivalent, 
+Confluent Platform 6.0) or newer.
+
+### Docker
+
+To run the `ksql-migrations` tool with docker, you may use either the ksqlDB 
+server or ksqlDB CLI image. Mount the root directory of your migrations project 
+into the container for use by the `ksql-migrations` tool. For example, the 
+following command creates a new migrations project in the local 
+`./my/migrations/dir` directory to connect to a ksqlDB server listening at 
+`http://localhost:8088` (which is accessed from within the docker container 
+at `http://host.docker.internal:8088`):
+
+```
+docker run -v $PWD/my/migrations/dir:/share/ksql-migrations confluentinc/ksqldb-server:{{ site.ksqldbversion }} ksql-migrations new-project /share/ksql-migrations http://host.docker.internal:8088
+```
+
+Similarly, the following initializes migrations metadata on the ksqlDB server 
+for the same setup:
+
+```
+docker run -v $PWD/my/migrations/dir:/share/ksql-migrations confluentinc/ksqldb-server:{{ site.ksqldbversion }} ksql-migrations --config-file /share/ksql-migrations/ksql-migrations.properties initialize-metadata
+```
+
+See the sections below for more on the different `ksql-migrations` commands.
+
+### Archive / deb / rpm
+
+TODO
```

Comment:
This will be added after the docs for archive / deb / rpm for standalone ksqlDB releases are added (cc @JimGalasyn do you have an update on this? Is it already in progress?)

Reply:
I know @andrewegel is working on this, but I don't know the status.

Reply:
Please translate my comments here: https://confluentinc.atlassian.net/browse/ST-4282?focusedCommentId=476050

into documentation.

Reply:
Thanks, Andrew. @JimGalasyn do you have a ticket for this? I think we'll want it for 0.16 even.

Reply:
Removed this TODO for now and filed a ticket (assigned to myself) to add it back in once the other docs for archive / deb / rpm packages are in.